### DISCUSSION TITLE
Stuck on Boot Menu. OpenCore is not loading.

### DISCUSSION THREAD

**Discussion by SaFamAnaK** [osy86/HaC-Mini#1051] [Q&A]
OpenCore is not loading. When getting into the Boot Menu I can see the USB drive but when pressing it nothing happens. I get a black screen for a second and then it gets back to the Boot Menu.
I did everything like it says in the guide so I expected it to load.

Env:
 - Running OSX version: Big Sur 11.1
 - Target drive OSX version: Monterey 12.5.1
 - HaC Mini installer version: 3.1.5

Thank you for the help!
[Installer.pdf](https://github.com/osy/HaC-Mini/files/9524748/Installer.pdf)

**Comment by Medizinmann-MD** [comment]
What is the currently installed Version of HaCMini.pkg?

Did you try erase NVRAM?

Did you try the guide for 3.0.1?

Did you try V3.2.0

**Comment by SaFamAnaK** [comment]
Hi, 
I haven't tried NVRAM or guide for 3.0.1 or 3.2.0. I will try now and update the results.  
The HaCMini.pkg version is 3.1.5

Thanks!

**Comment by SaFamAnaK** [comment]
I cannot erase the NVRAM because I cannot boot into OpenCore. 
When pressing on OpenCore nothing happens. 
I am adding an example. 
![WhatsApp Image 2022-09-08 at 8 23 26 PM](https://user-images.githubusercontent.com/97877416/189187305-e501bf05-d335-4497-ab35-cc2992e8b8be.jpeg)

**Comment by northeastwolf** [comment]
same issue，
I can't boot from the usb stick, i want install the Monterey for new nuc8i7hvk, but i can not use the OpenCore installer
and the usb stick can boot other computer, the opencore run normally on it

version:
HaC Mini installer version: 3.2.0
Monterey 12.5.1

**Comment by Medizinmann-MD** [comment]
> I cannot erase the NVRAM because I cannot boot into OpenCore. When pressing on OpenCore nothing happens. I am adding an example. ![WhatsApp Image 2022-09-08 at 8 23 26 PM](https://user-images.githubusercontent.com/97877416/189187305-e501bf05-d335-4497-ab35-cc2992e8b8be.jpeg)

I have actually never seen this menu...I boot via changing boot order in BIOS.
And maybe the Window Bootmanager interferes...looks like you are trinig dual boot? Never did this - I use one SSD for Windows and one for MacOS.

And the installer.pdf is the install.log from the installer?
Looks like the installer media creation failed!?

**Comment by Medizinmann-MD** [comment]
> same issue， I can't boot from the usb stick, i want install the Monterey for new nuc8i7hvk, but i can not use the OpenCore installer and the usb stick can boot other computer, the opencore run normally on it
> 
> version: HaC Mini installer version: 3.2.0 Monterey 12.5.1

Did you follow the instructions for the BIOS settings?
Did you use the correct Boot order?
How did you create the install media?

**Comment by northeastwolf** [comment]
> > same issue， I can't boot from the usb stick, i want install the Monterey for new nuc8i7hvk, but i can not use the OpenCore installer and the usb stick can boot other computer, the opencore run normally on it
> > version: HaC Mini installer version: 3.2.0 Monterey 12.5.1
> 
> Did you follow the instructions for the BIOS settings? Did you use the correct Boot order? How did you create the install media?

hi，thanks for reply
the bios settings is follow link: https://github.com/osy/HaC-Mini/blob/master/installation-guide/bios-settings.md#bios-settings
the installer create follow link: https://github.com/osy/HaC-Mini/blob/master/installation-guide/installation.md#creating-osx-installer
what's mean of the correct Boot order? i can boot the ubuntu and windows installer on it

**Comment by Medizinmann-MD** [comment]
Boot order - I mean „Boot Priority".
https://osy.gitbook.io/hac-mini-guide/installation-guide/installation#installing-macos

Okay - you can boot from USB sticks.

So it seems the installer creation might have failed...

You could try again and create a log - AFAIR there is an Option for creating an install.log.

Or - You could try to make a Catalina installer and then Upgrade through OTA-Update – that is how I did my last install two weeks ago.

**Comment by SaFamAnaK** [comment]
HI,
I just tried with Catalina as well and still the same. I downloaded HacMini.pkg 3.1.4 and tried to, with secure boot enabled and disabled and nothing.

**Comment by Medizinmann-MD** [comment]
> HI, I just tried with Catalina as well and still the same. I downloaded HacMini.pkg 3.1.4 and tried to, with secure boot enabled and disabled and nothing.

As I wrote before - I suspect your installer creation failed/failes!

Secure Boot must be disabled!

It should work - I used 3.2.0 with Catalina and it worked flawlessly.

**Comment by SaFamAnaK** [comment]
The installer creation seems ok. I get no errors. 
What is your BIOS version?

**Comment by SaFamAnaK** [comment]
Can you share some screenshots? I am sure I am doing it all right without getting ant errors.

**Comment by Medizinmann-MD** [comment]
I use the latest BIOS.
https://www.intel.de/content/www/de/de/download/19270/bios-update-hnkbli70.html

I will try to make a screen shot - may be to night or tomorrow night - HaC Mini isn't my primary machine.

**Comment by SaFamAnaK** [comment]
Thank you! Waiting for your update.

**Comment by SaFamAnaK** [comment]
@northeastwolf  Any luck?

**Comment by Medizinmann-MD** [comment]
I got bad news - while trying to redo the process and make some meaningful screenshots in the process I ran into a problem I didn't have before!

I can’t create a HaC mini installer USB anymore. The only thing that has changed is that my main HaC mini and my MacBook Pro are now on the latest Catalina. The HaCmini.pkg throws a 112 Error and fails.

The question is - can the installer still run successfully on the latest Catalina?

As I wrote - I tried two machines and both failed.

I will try it again on the other Hac Mini that runs with latest Monterey - but I don't have high hopes here...:-(

I then tried my “old” installer which I created two weeks ago with Catalina 15.7.03(before updating the MacBook) - and hitting F10 and starting OpenCore from this stick through the boot menu works.
This installer works in every way and I used it to install another NUC8i7HVK which I subsequently upgraded to Monterey 12.6..

**Comment by ivan-ef** [comment]
Same problem. I was able to install Catalina only one time. 
USB installer showed as OpenCore in bios, but "no bootable device detected" no matter what I do. 
Latest bios, bios reset. tried Catalina, Monterey install, HacMini.pkg 3.1.5, 3.2.0. installer creation has no errors (m1 MacBook Air if it matters).
BUT MY NUC DIDN'T BOOT FROM usb installer.

P.S. I formatted my flash-drive again and tried manually add clover bootloader. It still shows as efi usb opencore in bios. and still doesn't boot. Doesn't it's strange?

**Comment by juntian** [comment]
The same problem for me. Created the Ventura installer with Hacmini 3.2.2 package installed for the usb stick.  I can see the opencore in boot menu, but when choosing opencore boot, the screen will black and then return back to same the boot menu again.

**Comment by Genmer** [comment]
I have the same problem.
but I`m install macos13，use HaC Mini v3.2.2

**Comment by ivan-ef** [comment]
Looks like some boot files are stored in nvram, and it tries to boot from nvram no matter what  boot option you are choosing. Need to clear it first. Maybe cmos battery remove may help.

**Comment by Medizinmann-MD** [comment]
AFAIK removing CMOS battery won't help as NVRAM is stored in some kind of flash.

You could use some other working OpenCore to clear NVRAM.

**Reply by ivan-ef** [reply]
Yes you are right, nvram is not power dependent. Opencore usb drive is what I use. but it’s not always working because system that booting from a data stored in nvram is a vicious circle.

**Comment by ivan-ef** [comment]
I found a workaround. It’s possible to choose correct opencore boot record form built-in uefi shell. You need to run .efi file from correct fs*: volume.

**Reply by Medizinmann-MD** [reply]
Sounds great! Thanks for sharing the info!

**Reply by farmerpaul** [reply]
Interesting. Can you share what command I would have to type in the UEFI shell? I'm also banging my head against the wall trying to get OpenCore to boot.

**Reply by ivan-ef** [reply]
> Interesting. Can you share what command I would have to type in the UEFI shell? I'm also banging my head against the wall trying to get OpenCore to boot.

fs0: 
(usually it's first disk partition with uefi boot record. may vary on your setup )

cd EFI/OC/
(opens directory with Opencore bootloader)

OpenCore.efi
(runs bootloader)

you can use "dir" to see what files are in you current directory.

after running opencore you may need to reset nvram to make opencore default boot record

**Reply by farmerpaul** [reply]
Thanks, I did actually get it figured out!! 😌

**Comment by juntian** [comment]
I found out a workaround for the same problem in my environment caused by wrong info in the NVRAM of the NUC. I removed the useless boot options in NVRAM by bcdedit in Windows (I have another disk w/ Windows installed can be booted). For Linux and UEFI shell, there are similar commands. Linux can use efibootmgr, efi shell can use bcfg.

   bcdedit /enum firmware  # show the boot list, w/ lots of duplicated or useless options like opencore usb boot options, old linux boot menu.
   bcdedit /delete {the_useless_option_UUID} # remove the useless boot option one by one

Reboot system then I can boot the right opencore installer now.

In efi shell, guys told there are bcfg command to remove the boot option, but in NUC default efi shell. I didn't find out it. Therefore I used windows bcdedit tool in CLI.

**Reply by Medizinmann-MD** [reply]
One could also try a GUI-UEFI-Editor i.e.:

EasyBCD
https://neosmart.net/EasyBCD/

or EasyUEFI
https://www.easyuefi.com/index-us.html
